src/cloudflare/internal/d1-api.ts

```diff
@@ -42,6 +63,7 @@ class D1Database {
     return new D1PreparedStatement(this, query)
   }
 
+  // DEPRECATED, TO BE REMOVED WITH NEXT BREAKING CHANGE
```

Comment:
IMO `dump()` is safe to remove today, since it's a no-op for SRS-backed dbs

Reply:
Yeah I know but it's still a breaking change unless we've already shut down all alpha DBs. So I was going to leave it for now and fix it when we're forced to introduce a compat date.